docusaurus/docs/guides/deployments/10-linux/20-router/30-configuration.md

```diff
@@ -28,7 +28,7 @@ identity:
 
 ### Private Key
 
-The shape of the [`identity`](/reference/30-configuration/conventions.md#identity) section is conventional for routers and controllers, but the `key` sub-property behaves differently for routers. If a private key exists in the path, then it will be used to sign router enrollment certificate requests. If it does not exist, then a private key will be locally generated and stored in that path.
+The [`identity`](/reference/30-configuration/conventions.md#identity) configuration section is conventional for routers and controllers, but the `key` sub-property behaves differently for routers. If a private key exists in the path, then it will be used to sign router enrollment certificate requests. If it does not exist, then a private key will be locally generated and stored in that path.
```

Comment:
the changes above through here all still reference-esque and not "guide" esque imo.

Reply:
> configuration section is conventional for routers and controllers

"conventional" is too formal and not conversational and sounds/reads awkwardly to me. it should be replaced with "common" or anything less clinical imo

Reply:
Improved in https://github.com/openziti/ziti-doc/pull/868